docs/reference/index-modules/allocation/delayed.asciidoc

```diff
@@ -0,0 +1,89 @@
+[[delayed-allocation]]
+=== Delaying allocation when a node leaves
+
+When a node leaves the cluster for whatever reason, intentional or otherwise,
+the master reacts by:
+
+* Promoting a replica shard to primary to replace any primaries that were on the node.
+* Allocating replica shards to replace the missing replicas (assuming there are enough nodes).
+* Rebalancing shards evenly across the remaining nodes.
+
+These actions are intended to protect the cluster against data loss by
+ensuring that every shard is fully replicated as soon as possible.
+
+This ``shard-shuffle'' can, however, put a lot of extra load on the cluster
```

Comment:
I would explain that we do throttling, something like "Even though throttling is done on a node level for concurrent recoveries, as well as across the cluster, this "shard shuffle" can cause extra load on ...."